frontend/workflows-lib/lib/components/template/TemplateCard.tsx

```diff
@@ -34,22 +34,40 @@ export const TemplateCard: React.FC<TemplateCardProps> = ({ template }) => {
             }}
           >
             <CardContent>
-              <Box
-                display="flex"
-                justifyContent="space-between"
-                alignItems="center"
-              >
-                <Typography gutterBottom variant="h5" component="div">
-                  {template.title ?? template.name}
-                </Typography>
-                <Typography variant="body2" sx={{ color: "text.secondary" }}>
-                  Maintainer: {template.maintainer}
-                </Typography>
-              </Box>
-              <Typography gutterBottom>{template.name}</Typography>
-              <Typography variant="caption" gutterBottom>
-                {template.description ?? ""}
-              </Typography>
+              <Stack spacing={1}>
+                <Box
+                  display="flex"
+                  justifyContent="space-between"
+                  alignItems="center"
+                >
+                  <Typography variant="h5">
+                    {template.title ?? template.name}
+                  </Typography>
+                  <Typography variant="body2" sx={{ color: "text.secondary" }}>
+                    Maintainer: {template.maintainer}
+                  </Typography>
+                </Box>
+                <Box
+                  display="flex"
+                  justifyContent="space-between"
+                  alignItems="center"
+                >
+                  <Typography>{template.name}</Typography>
+                  {template.repository && (
+                    <Typography
+                      variant="body2"
+                      sx={{ color: "text.secondary" }}
+                    >
+                      Repository: {template.repository}
+                    </Typography>
+                  )}
+                </Box>
+                {template.description && (
+                  <Typography variant="caption">
+                    {template.description}
```

Comment:
template.description lost it's fallback string here. I don't know if that's intentional or significant?